assets/model_monitoring/components/src/shared_utilities/store_url.py

```diff
@@ -91,7 +92,8 @@ def valid_aml_obo_credential():
                 aml_obo_credential = AzureMLOnBehalfOfCredential()
                 if validate_aml_obo_credential:
                     try:
-                        _ = aml_obo_credential.get_token()
+                        _ = aml_obo_credential.get_token("https://management.azure.com/.default")
```

Comment:
do we need the /.default for the scope?

Reply:
I think either should be fine. I saw most codes in this repo has the /.default, so just keep for consistency

https://github.com/search?q=repo%3AAzure%2Fazureml-assets+get_token%28&type=code